lib/oper1.g

```diff
@@ -745,7 +745,7 @@ InstallAttributeFunction(
 ##  <#/GAPDoc>
 ##
 IsPrimeInt := "2b defined";
-
+MakeWriteOnceAtomic := "2b defined";
```

Comment:
@rbehrends do you think we may read MakeWriteOnceAtomic earlier, or "2b defined" is the only way to go here?